enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/ExpressionAcceptanceTest.scala

```diff
@@ -97,15 +97,15 @@ class ExpressionAcceptanceTest extends ExecutionEngineFunSuite with CypherCompar
   test("prepending item to a list should behave correctly in all runtimes") {
     val query = "CYPHER WITH {a:[1,2,3]} AS x RETURN 'a:' + x.a AS r"
 
-    val result = succeedWith(Configs.All - Configs.EnterpriseInterpreted, query)
+    val result = succeedWith(Configs.AllExceptSleipnir, query)
```

Comment:
I want a PR that causes as little conflict with existing PR's, while still also achieving the goal of the PR. I drew the line at renaming internal classes that are commonly touched by ongoing work. I think phasing this in two phases will make this much easier.